dataops-salesforce/src/main/scala/notebook/dataops-excel.scala

```diff
@@ -0,0 +1,109 @@
+import org.apache.spark.sql.{DataFrame,SparkSession}
```

Comment:
This file shouldn't be in this location.

The folder path for this file should be:
dataops-excel/src/main/scala/notebook/dataops-excel.scala